heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -48,15 +50,13 @@
 import com.spotify.heroic.querylogging.QueryLoggerFactory;
 import com.spotify.heroic.statistics.DataInMemoryReporter;
 import com.spotify.heroic.statistics.MetricBackendReporter;
+
```

Comment:
This indicates that the wrong formatter is in effect for imports, we should no longer be adding spaces between different groups of imports other than **static** and **regular**. Look into updating yours.